packages/backend/src/accounting/psql/ledger-transfer/index.ts

```diff
@@ -60,21 +62,42 @@ export async function getAccountTransfers(
             )
         )
     )
+  if (limit) builder.limit(limit)
 
-  return transfers.reduce(
+  return (await builder).reduce(
     (results, transfer) => {
-      if (transfer.debitAccountId === accountId) {
-        results.debits.push(transfer)
+      if (transfer.debitAccountId === id) {
+        results.debits.push(mapToGetLedgerTransfersResult(transfer))
       } else {
-        results.credits.push(transfer)
+        results.credits.push(mapToGetLedgerTransfersResult(transfer))
       }
-
       return results
     },
-    { credits: [], debits: [] } as GetTransfersResult
+    { credits: [], debits: [] } as GetLedgerTransfersResult
   )
 }
 
+function mapToGetLedgerTransfersResult(
+  dbModel: LedgerTransfer
+): ServiceLedgerTransfer {
+  return {
+    amount: dbModel.amount,
+    creditAccountId: dbModel.creditAccount ? dbModel.creditAccount.id : '',
+    debitAccountId: dbModel.debitAccountId,
+    id: dbModel.id,
+    ledger: dbModel.ledger,
+    timestamp: BigInt(dbModel.createdAt.getTime()),
+    type:
+      dbModel.type === LedgerTransferType.WITHDRAWAL
+        ? TransferType.WITHDRAWAL
+        : TransferType.DEPOSIT,
```

Comment:
I _think_ we don't end up passing through the `TransferType.TRANSFER` for normal account to account transfers for the PSQL accounting service. We should, otherwise this will mark "normal" transfers as `TransferType.DEPOSIT`.